src/pymap/management/commands/create_management_groups.py

```
@@ -0,0 +1,76 @@
+from migrator.models import CeleryTask
+
+from django.db.models import Model
+
+from django_celery_results.models import TaskResult
+
+
+class Command(BaseCommand):
+    help = "Adds a group for managing the users trough the admin dashboard"
+
+    def manage_group(
+        self, group_name: str, model: type[Model]|None=None, group_permissions: list[str] = []
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)